index.html

```diff
@@ -2,11 +2,520 @@
+    <img src="images/10diary-illos-04-threeByTwoSmallAt2X.jpg" alt="">
+    <div class="sp-as-items-desc">
+      <span class="img-title-as"></span>
+      <p><h3>‘I Watched People Pass My Booth and Listened to Scraps of Conversation’</h3></p>
+      <span class="space-date">Jan. 10</span>
+    </div>
+  </div>
+  
+    <div class="sp-as-items">
+    <img src="images/06Techfix-illo-threeByTwoSmallAt2X.jpg" alt="">
+    <div class="sp-as-items-desc">
+      <span class="img-title-as">Glenn Harvey</span>
+      <p><h3>The Tech That Will Invade Our Lives in 2021</h3></p>
+      <span class="space-date">Jan. 6</span>
+    </div>
+  </div>
+  </div>
+
+  </section>
+  <section class="popular-letters">
+    <div class="popular-letter-head">
+      <h2>Most Popular</h2>
+     </div>
+    <div>
+     <ul>
+       <li>One Division, Seven Canadian Teams and a Lot of Travel</li>
+       <li>Why a Vogue Cover Created an Uproar Over Kamala Harris</li>
+       <li>A Colonoscopy Alternative Comes Home</li>
+       <li>Should I Warn My Ex’s Fiancée About His Cheating Heart?</li>
+       <li>Normally Sharp, He Was Chronically Confused. What Was Going On?</li>
+       <li>Opinion: When an I.Q. Score Is a Death Sentence</li>
+       <li>How the Right Foods May Lead to a Healthier Gut, and Better Health</li>
+       <li>On the Way Out, Melania Trump Denounces Attacks on Her as ‘Shameful’</li>
+       <li>The Attack on the Capitol Was Even Worse Than It Looked</li>
+       
+       
+     </ul>
+      
+    </div>
+  </section>
+  </div>
+  <div class="footer-tab">
+    <div class="title-image">
+     <img class="small-title" src="https://theme.zdassets.com/theme_assets/968999/3f343ffc604a6c839e31d3f052b78abfa34ab588.png" alt="the New York Times">
+     <div class="other-button">
+       <a class="go-to-homepage" href="#">Go to Home Page »</a>
+     </div>
+    </div>
+    <div class="lower-list">
+    <ul>
+      <li>NEWS</li>
+      <li>OPINION</li>
+      <li>ARTS</li>
+      <li>LIVING</li>
+      <li>LISTINGS & MORE</li>
+    </ul>
+  </div>
+</div>
+
+<div class="footer-desktop">
+  <div class="title-image">
+   <img class="small-title" src="https://theme.zdassets.com/theme_assets/968999/3f343ffc604a6c839e31d3f052b78abfa34ab588.png" alt="the New York Times">
+   <div class="other-button">
+     <a class="go-to-homepage" href="#">Go to Home Page »</a>
+   </div>
+  </div>
+  <div class="footer-desktop-cols">
+ <div><ul>
+   <h4>NEWS</h4>
+   <li><a href="#">Home Page</a></li>
+   <li><a href="#">World</a></li>
+   <li><a href="#">Coronavirus</a></li>
+   <li><a href="#">U.S.</a></li>
+   <li><a href="#">Politics</a></li>
+   <li><a href="#">Election Results</a></li>
+   <li><a href="#">Newyork</a></li>
+   <li><a href="#">Business</a></li>
+   <li><a href="#">Tech</a></li>
+   <li><a href="#">Science</a></li>
+   <li><a href="#">Obtituaries</a></li>
+   <li><a href="#">Today's Paper</a></li>
+   <li><a href="#">Corrections</a></li>
```

Comment:
Fix this links

Reply:
Hello. I have fixed the points you mentioned. Could you please check it?